cmd/list.go

```diff
@@ -15,7 +15,12 @@ var listCmd = &cobra.Command{
 	Use:   "list",
 	Short: "List wallets or accounts within a wallet",
 	Run: func(cmd *cobra.Command, args []string) {
-		if walletIndex < 0 {
+		if walletAccount != "" {
+			// For when a specific account is specified. A single account is returned.
+			rpcClient := rpc.Client{URL: rpcURL}
```

Comment:
Can this line be moved into getBalanceAndPrint?

Reply:
Done!

Reply:
@hectorchu by the way, do you have Discord? Would it be good to setup a Discord channel for gonano quick support and discussions when other people want to contribute?

Reply:
I think it's better to keep it all in GitHub.

Reply:
Ok, np. What is the best way to chat about new contributions to gonano?

Please check commit c33c60d

Reply:
By opening a new issue.